common/djangoapps/util/help_context_processor.py

```diff
@@ -54,27 +54,78 @@ def get_config_value_with_default(section_name, option, default_option="default"
                               section_name, option)
             return config_file_object.get(section_name, default_option)
 
+        def get_config_value_with_override_section(section_name, option, override_section_name):
+            """
+            Args:
+                section_name: name of the section in the configuration from which the option should be found
+                option: name of the configuration option
+                override_section: name of the section in the configuration that may contain an overriding property
+            """
+            if option:
+                try:
+                    # If the property is in the override section, use its value.
+                    return config_file_object.get(override_section_name, option)
+                except (ConfigParser.NoOptionError, AttributeError):
+                    # If the property is not in the override section, get its
+                    # value from the primary configuration section.
+                    return get_config_value_with_default(section_name, option)
+
         def get_doc_url():
             """
             Returns:
                 The URL for the documentation
             """
+            # If the LMS is used for edx.org, return documentation URLs
+            # for edX partner documentation. If not, return Open edX
+            # documentation. Base URLs will always be different. The
+            # document path might be different.
+            if settings.USE_EDX_PARTNER_DOCUMENTATION:
+                doc_base_url = get_config_value_with_override_section(
```

Comment:
it seems like you could pull this variable right out of configuration, without needing to build the overrides ini file.   This would be preferable to having hard-coded edx.org related URLs in the openedx platform.